sdk/core/core/src/az_http_policy.c

```diff
@@ -5,12 +5,16 @@
 #include <az_http_pipeline.h>
 #include <az_http_policy.h>
 #include <az_http_request_builder.h>
+#include <az_log.h>
 #include <az_mut_span.h>
 #include <az_span.h>
 #include <az_span_builder.h>
 #include <az_str.h>
 #include <az_url.h>
 
+#include <stddef.h>
+#include <time.h>
```

Comment:
You may not need this if we get rid of clock_t

Reply:
I think we are going to need it in the implementation - even though users will be specifying threshold in milliseconds, the time we get from `clock()` is in `clock_t` and there is `CLOCKS_PER_SEC` for me to do the conversions necessary.

Reply:
Done.